src/constants/feature.js

```diff
@@ -36,6 +36,7 @@ export default {
   TextSelection: 'TextSelection',
   TouchScrollLock: 'TouchScrollLock',
   Copy: 'Copy',
+  MultipleViewerMerging: 'MultipleViewerMerging',
```

Comment:
I think we need to have some documentation about this feature.

Reply:
Thanks for the reminder. I added a comment to the top of this page